main.tf

```diff
@@ -0,0 +1,36 @@
+terraform {
```

Comment:
Nitpick: This looks like tf 0.11 syntax which is OK but (as an improvement) we could use 0.12.
Though I'd also recommend using `tfenv` for pinning code to specific version of terraform which is not always backward compatible.
[1] https://github.com/tfutils/tfenv